cpp/include/cudf/io/csv.hpp

```diff
@@ -1339,7 +1339,7 @@ class csv_writer_options {
   // string to use for values == 0 in INT8 types (default 'false')
   std::string _false_value = std::string{"false"};
   // Optional associated metadata
-  table_metadata const* _metadata = nullptr;
+  std::vector<std::string> _names;
```

Comment:
Sorry I still missed doxygen for this variable too. Probably the last missed one :smile: